### DISCUSSION TITLE
Next Virtual OpenDCS Workshop 11/1/2022

### DISCUSSION THREAD

**Discussion by NathanHolcomb** [opendcs/opendcs#53] [General]
Based on initial feedback, I am proposing Tuesday, November 1, 2022 for the next OpenDCS Virtual Workshop. To allow for people from both coasts to attend I believe a time slot from 11am - 5pm Eastern would work best. 

Please use this discussion area for proposing any topics, enhancements, presentations, or general discussions you would like to see during the Workshop.

**Comment by MikeNeilson** [comment]
We'll definitely want a review of how the shared development is going and lessons learned.

**Comment by agilmore2** [comment]
Date in the title says November 2. Description says Nov 1. Which is it?

**Reply by NathanHolcomb** [reply]
Thanks and sorry for the fat fingers.

**Comment by NathanHolcomb** [comment]
We are less than a month away from the proposed virtual OpenDCS Workshop on 11/1/22. I just wanted to first confirm that folks were still interested in having a virtual meet up (I personally think it would be a good check in) and if so, what topics we want to address or cover during the workshop. 

Please use this thread to make any recommendations for the virtual meeting as well as what you would like to see presented or discussed.

Thanks!

**Comment by MikeNeilson** [comment]
For me

- Yet more dev topics, future designs and work. (at least knowing if people are okay with me just going forward on things discussed.)
- some sort of official charter.
- Identification from community of critical needs.
